pkg/controller/planexecution/planexecution_controller.go

```diff
@@ -621,3 +622,8 @@ func (r *ReconcilePlanExecution) Cleanup(obj runtime.Object) error {
 
 	return nil
 }
+
```

Comment:
This is unrelated to the bug, just making our lives easier when debugging integration tests.